jenkins/spark-premerge-build.sh

```diff
@@ -124,6 +126,10 @@ ci_2() {
     TEST='not conditionals_test and not window_function_test and not struct_test and not time_window_test' \
       ./integration_tests/run_pyspark_from_build.sh
     INCLUDE_SPARK_AVRO_JAR=true TEST='avro_test.py' ./integration_tests/run_pyspark_from_build.sh
+    # export 'LC_ALL' to set locale with UTF-8 so regular expressions are enabled
+    LC_ALL="en_US.UTF-8" TEST="regexp_test.py" ./integration_tests/run_pyspark_from_build.sh
+    # export 'LC_ALL' to set locale without UTF-8 so regular expressions are disabled to test fallback
+    LC_ALL="en_US.iso88591" TEST="regexp_no_unicode_test.py" ./integration_tests/run_pyspark_from_build.sh
```

Comment:
looks like this one should already be covered in normal tests run w/o explicitly setting LC_ALL?

Reply:
removed this redundant last line in last commit (it was artifact of the previous code organization)